R/students.R

```diff
@@ -1286,7 +1297,8 @@ s_29 <- function(input_df=usheUtils::fake_student_df) {
 
   output_df <- input_df %>%
     # Calculate intermediate fields
-    mutate( s_hb75_waiver = house_bill_75_waiver ) %>%
+    #mutate( s_hb75_waiver = house_bill_75_waiver ) %>%
+    mutate( s_hb75_waiver = if_else(house_bill_75_waiver > 100, 100, house_bill_75_waiver) ) %>%
```

Comment:
@andreabr-dixie @jpfeifer-dsu 
There are also negative values in this data field. I think we might need to review how this data field is being created in Edify.

Perhaps also specify that it is a percentage based field?